python/lsst/rubintv/production/starTracker.py

```diff
@@ -873,6 +878,7 @@ def catchupCamera(self, camera, dayObs):
                                                 cameraType=camera.cameraType)
         for file in filenames:
             self.log.info(f'Catching up {file}')
+            gc.collect()
```

Comment:
Is this still necessary, after the matplotlib `plot` changes?

Reply:
You know, probably not, but I don't think it does any harm. Then again, I do dislike code which scatters those around for no reason, so maybe I will removed and see if it's all OK with it actually, I can always put it back, but if I leave it it'll likely live on forever.